### PR TITLE
Updated pure getter properties to style guidelines

### PR DESCRIPTION
**Pull request by marcusnaslund**
Peer review @sebastianbaginski or @thomasfanell ?

### CODE REVIEW

**Comment by sebastianbaginski** [comment]
It looks OK.

File: source/sdk/io/File.ooc [hunk 1]
```diff
@@ -25,11 +25,9 @@ File: abstract class {
 	extension: String { get { getExtension() } }
 	nameWithoutExtension: String { get { getNameWithoutExtension() } }
 	pathWithoutExtension: String { get { getPathWithoutExtension() } }
-	children: VectorList<This> {
-		get {
-			getChildren()
-		}
-	}
+	children: VectorList<This> { get {
+		getChildren()
```
Comment (sebastianbaginski) [inline source/sdk/io/File.ooc]: `This` or `this` ?

Comment (marcusnaslund) [inline source/sdk/io/File.ooc]: Yes, but that's missing in 1000 other places in this file. We want to enforce `this`, so I will be adding it in some future PR. For now, this does exactly what it used to do.

Comment (sebastianbaginski) [inline source/sdk/io/File.ooc]: OK

**Pull request merged**